### PR TITLE
Add update command with & Docker script in cloud-storage module

### PR DESCRIPTION
**Pull request by emgaurv**
I've added feature in clocal-gcp, cloud-storage module cli to update object &  script to run this command inside the docker container.

### CODE REVIEW

File: src/services/cli-commands/cloud-storage/cmd.js [hunk 1]
```diff
@@ -63,7 +65,20 @@ const _uploadObject = (srcPath, dockerId, destPath) => {
...
+        if (err) console.log(chalk.bgRed(`failed to execute\n${stderr}`));
+        console.log(`${stdout}`);
+      }
+    );
+    throw Error('not implemented yet');
+  } catch (err) {
+    return false;
+  }
+};
 const _downloadObject = (srcPath, dockerId, destPath) => {
```
Comment (rehrumesh) [inline src/services/cli-commands/cloud-storage/cmd.js]: Add a blank line here.

File: src/services/cli-commands/cloud-storage/cmd.js [hunk 2]
```diff
@@ -63,7 +65,20 @@ const _uploadObject = (srcPath, dockerId, destPath) => {
     );
   });
 };
-
+const updateObject = (bucketName) => {
```
Comment (rehrumesh) [inline src/services/cli-commands/cloud-storage/cmd.js]: Add a blank line above.

**Comment by hcktheheaven** [comment]
@rehrumesh  Done!

**Comment by dilantha111** [comment]
Hi @hcktheheaven  we can only see the command part. Update script is not there in the PR.

**Pull request closed without merging**